src/drivers/MAVLinkServer/MAVProxy/uav_viewer_py.yml

```diff
@@ -1,30 +1,30 @@
 Camera:
-  Server: 1 # 0 -> Deactivate, 1 -> Ice , 2 -> ROS
+  Server: "Ice" # Deactivate, Ice , ROS
```

Comment:
We prefer not to have quotes. As easy as possible for the end user :D ```Server: "Ice"``` --> ```Server: Ice```
Also, this should be case insensitive (ice, ICE, IcE, Ice, ROS, ros, RoS, rOS, ...)

Please, change this in every yml file!